Recrutify/Recrutify.Host/Extensions/MiddlewareExtensions.cs

```diff
@@ -9,5 +10,12 @@ public static void UseHttpStatusExceptionHandler(this IApplicationBuilder app)
         {
             app.UseMiddleware<ExceptionMiddleware>();
         }
+
+        public static void RegistrationStatusEvent(this IApplicationBuilder app)
```

Comment:
registration of middleware usually begins with Use
so write UseStatusCnageEventProcessor